core/src/main/scala/commandcenter/command/StocksCommand.scala

```diff
@@ -0,0 +1,70 @@
+package commandcenter.command
+
+import com.typesafe.config.Config
+import commandcenter.CCRuntime.Env
+import commandcenter.command.StocksCommand.StocksResult
+import commandcenter.view.DefaultView
+import io.circe.{ Decoder, Json }
+import sttp.client.circe.asJson
+import sttp.client.httpclient.zio.SttpClient
+import sttp.client.{ basicRequest, UriContext }
+import zio.{ IO, TaskManaged, ZIO, ZManaged }
+
+final case class StocksCommand(commandNames: List[String], tickers: List[Ticker]) extends Command[Unit] {
+  val commandType: CommandType = CommandType.StocksCommand
+  val title: String            = "Stock"
+  val stocksUrl: String        = "https://query1.finance.yahoo.com/v7/finance/quote?symbols="
+
+  def preview(searchInput: SearchInput): ZIO[Env, CommandError, List[PreviewResult[Unit]]] =
+    for {
+      input    <- ZIO.fromOption(searchInput.asPrefixed).orElseFail(CommandError.NotApplicable)
+      symbols   = if (input.rest.isBlank) tickers.map(_.id).mkString(",") else input.rest
+      request   = basicRequest
+                    .get(uri"$stocksUrl$symbols")
+                    .response(asJson[Json])
+      response <- SttpClient
+                    .send(request)
+                    .map(_.body)
+                    .absolve
+                    .mapError(CommandError.UnexpectedException)
+      stocks   <- IO.fromEither(
+                    response.hcursor.downField("quoteResponse").downField("result").as[List[StocksResult]]
+                  ).mapError(CommandError.UnexpectedException)
+    } yield stocks.map { stock =>
+      Preview.unit
+        .score(Scores.high(input.context))
+        .view(
+          DefaultView(
+            title,
+            fansi.Str.join(
+              fansi.Color.Cyan(stock.ticker),
+              fansi.Str(s" (${stock.name}) "),
+              fansi.Color.LightCyan(String.format("%.2f", stock.price))
```

Comment:
Since it's fairly standard, I would also append the last daily change (and maybe the currency too) like `441.50 USD +20.79 (4.95%)`.

It looks like the fields for this are `regularMarketChange`, `regularMarketChangePercent`, and `financialCurrency` in the API response.

If we're going with that, then separate colors for gains vs. losses would also make sense. The colors are actually cultural-specific for this. Gain=green, Loss=red in most Western countries I believe, but it's the exact opposite in Japan, China, etc. You don't have to worry about handling all the different cultures though. That might be a bit much. I can always add that part in later.